src/worker.ts

```diff
@@ -172,7 +172,7 @@ function onMessage (
         result: null,
         // It may be worth taking a look at the error cloning algorithm we
         // use in Node.js core here, it's quite a bit more flexible
-        error
+        error: error instanceof Error ? error : null
```

Comment:
Why this change?

Reply:
@metcoder95 checking the linting errors.

Reply:
LGTM 👍

Reply:
OK. So the linting error is:
```bash
/r/piscina/test/test-uncaught-exception-from-handler.ts
  0:0  error  Parsing error: DeprecationError: 'originalKeywordKind' has been deprecated since v5.0.0 and can no longer be used. Use 'identifierToKeywordKind(identifier)' instead
```
And it appears on the very top of, pretty much, every `ts` file.

Now I used the `rg --no-ignore originalKeywordKind` to look for that keyword and it only appears in `node_modules` packages related to typescript and lining. And not a single occurrence in the `src` folder.

My best guess here it is a configuration and compatibility issue between `ts 5.x.x.` and linter. Since this package is using `standardx`, then the underlying `eslint` version is `7` (the oldish `standard` dependency). I would suggest updating the `standardx` dependency once it updates its own dependency. There is a PR for that: https://github.com/standard/standardx/pull/42 - however it is stale for some time.

I tried few different package versions of typescript and linter plugins, also tried to do `npm override` on the `eslint`
package version - but ran into breaking changes issue.

Reply:
I made some more changes to linter plugin versions and version of typescript (still 5th as major) and that has gotten rid of the linter errors locally.